src/test/java/io/appform/jsonrules/DiffFromCurrentEpochOperationTest.java

```diff
@@ -0,0 +1,99 @@
+package io.appform.jsonrules;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.appform.jsonrules.expressions.composite.NotExpression;
+import io.appform.jsonrules.expressions.composite.OrExpression;
+import io.appform.jsonrules.expressions.equality.EqualsExpression;
+import io.appform.jsonrules.expressions.numeric.GreaterThanExpression;
+import io.appform.jsonrules.expressions.numeric.LessThanExpression;
+import io.appform.jsonrules.expressions.preoperation.date.DiffFromCurrentEpochOperation;
+import io.appform.jsonrules.utils.Rule;
+import io.appform.jsonrules.utils.TestUtils;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.util.Collections;
+
+/**
+ * Created by shubham.ppe on 25/07/17.
+ */
+public class DiffFromCurrentEpochOperationTest {
+
+    private ExpressionEvaluationContext context;
+    private ObjectMapper mapper;
+    private Instant dateTime;
+
+    @Before
+    public void setUp() throws Exception {
+        mapper = new ObjectMapper();
+        dateTime = Instant.now();
+        long epoch = dateTime.getEpochSecond();
```

Comment:
We don't use seconds anywhere. Epoch naturally come in milliseconds. Change code to reflect the same everywhere.